metadata.json

```diff
@@ -1,15 +1,9 @@
 {
     "description": "Adds a button to the panel that shows information Spotify playback. For bug reports, feature requests, translation contributions, etc., please visit the extension's github page.",
     "name": "spotify-tray",
-    "shell-version": [
-        "40",
-        "41",
-        "42",
-        "43",
-        "44"
-    ],
+    "shell-version": ["45"],
```

Comment:
Note that all the previous versions were dropped. The docs mention explicitely that v45 is not compatible with older versions due to the changed imports syntax. 

See more: https://release.gnome.org/45/developers/index.html